src/jaxsim/parsers/descriptions/collision.py

```diff
@@ -119,3 +119,16 @@ class SphereCollision(CollisionShape):
 
     def __eq__(self, other):
         return (self.center == other.center).all() and super().__eq__(other)
+
+
+@dataclasses.dataclass
+class MeshCollision(CollisionShape):
+
+    center: npt.NDArray[np.float64]
```

Comment:
Let's keep this simple, we never use explicit types throughout our codebase.

```suggestion
    center: npt.NDArray
```